src/services/post.service.ts

```diff
@@ -12,6 +13,18 @@ class PostService{
     const posted: BasePostDto = await pgDataSource.manager.save(post);
     return posted;
   }
+
+  public async updatePost(postData: BasePostDto, userId: number) : Promise<BasePostDto> {
+    const currentPost = postData.id
+    const currentUser = userId
+    const findPost = await PostEntity.findOne({where:{id:currentPost, userId: currentUser}})
```

Comment:
Don't need userId: currentUser. 
We can have these conditions.
`if (!findPost) throw new HttpException(404,'Post Not Found')
`if(findPost.userId !== currentUser) throw new HttpException(401, 'Unauthorized to make updates to this post')`
`